pkg/config/types.go

```diff
@@ -30,8 +30,19 @@ const (
 
 var _ Type = SettingsType{}
 
+type AllUserPermissionKind = string
+
+const (
+	Read  AllUserPermissionKind = "read"
```

Comment:
Should we maybe prefix this one? When using `config.Read` this is missing some context